README.md

```diff
@@ -1,46 +1,35 @@
-# Getting Started with Create React App
+# 🚴‍♀️ Plova 🚴‍♂️
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+[> Plova web site](http://nodeca.github.io/pica/demo/ "title text!")
 
-## Available Scripts
+```
+Demo version of famous delivery application 'Glovo'
+```
 
-In the project directory, you can run:
+App presentation using development best practices like:
 
-### `npm start`
+1. React & TypeScrypt
```

Comment:
Always check for typos. Use spellchecking plugin for VS Code.

Reply:
Thanks. I will.